API/src/microservices/Assessment-Service/index.js

```diff
@@ -0,0 +1,94 @@
+const Assessment = require("../Database/assessment");
+const bookshelf = require("../Database/bookshelf");
+
+//TODO: update created_at, convert Date to prostgreSQL date object
+
+//assessment object
+let assessment = {};
+
+//formats data for database and calculates the assessment score and risk levels
+function prepareData(data) {
+  assessment = {
+    name: data.name,
+    instrument: data.instrument,
+    altercations: parseInt(data.altercations),
+    previousContact: parseInt(data.previousContact),
+    ownerAltercation: parseInt(data.ownerAltercation),
+    playWellDogs: parseInt(data.playWellDogs),
+    hissesStrangers: parseInt(data.hissesStrangers),
+    birthday: data.birthday,
+  };
+  calculateScore(assessment);
+  calculateRiskLevel(assessment);
+}
```

Comment:
```suggestion
  calculateRiskLevel(assessment);
}

return assessment;
```

Reply:
something like this.  So you'd pass the assessment to each function and then return it to the route when it's finished, but we keep the variables locally defined